frappe/public/scss/desk/list.scss

```diff
@@ -81,6 +81,29 @@
 	@include get_textstyle("base", "regular");
 	border-radius: var(--border-radius-md);
 
+	@media (max-width: 768px) {
+		height: fit-content;
+		padding-top: 4.5px;
+		padding-bottom: 4.5px;
+
```

Comment:
let's decrease font size as well ?
```suggestion
		@include get_textstyle("2xs", "regular");

```

Reply:
It feels a little too small to me:
<img width="377" alt="image" src="https://github.com/frappe/frappe/assets/62411302/c4e42579-a463-48c9-9724-a598852c9d80">

What about `md`? It's 13px, while this is 12. Here, the subject is `md` while the bottom is `sm`.
<img width="397" alt="image" src="https://github.com/frappe/frappe/assets/62411302/3066c99a-c5a1-43fc-b344-7e0c6dbe351a">